app/javascript/mastodon/service_worker/entry.js

```diff
@@ -1 +1,10 @@
 import './web_push_notifications';
+
+// Cause a new version of a registered Service Worker to replace an existing one
+// that is already installed, and replace the currently active worker on open pages.
+self.addEventListener('install', function(event) {
```

Comment:
@nolan Am I right in assuming that this is safe to do since we do not cache and there is no communication between the clients and the Service Worker?